src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -78,6 +78,7 @@ public class AccordService implements IAccordService, Shutdownable
     private final AccordConfigurationService configService;
     private final AccordScheduler scheduler;
     private final AccordVerbHandler<? extends Request> verbHandler;
+    private final AccordJournal journal;
```

Comment:
do we need this reference?  think we can drop as this is *owned* by `CommandStores`?

Reply:
Dropped.